ultralytics/cfg/__init__.py

```diff
@@ -31,27 +31,30 @@
 
 # Define valid tasks and modes
 MODES = {"train", "val", "predict", "export", "track", "benchmark"}
-TASKS = {"detect", "segment", "classify", "pose", "obb"}
+TASKS = {"detect", "segment", "classify", "pose", "obb", "human"}
 TASK2DATA = {
     "detect": "coco8.yaml",
     "segment": "coco8-seg.yaml",
     "classify": "imagenet10",
     "pose": "coco8-pose.yaml",
     "obb": "dota8.yaml",
+    "human": "human8.yaml",
```

Comment:
Is human8.yaml based on COCO? Maybe we should call it coco8-human.yaml to align with the seg and pose yaml naming convention we already have?

Reply:
Updated!